File: Python/Printcharacter.py
# Define modules if needed

# Python does not need an entry point below is the main

# if __name__ == "__main__":
# execute only if run as a script
# main()

# Ask how many A's should be written
# and the name of the file

print("How many 'A' should be written to a file: ")
num = input()

print("What is the name for the file: ")
filename = input()

# Create a file and write to it then close it
with open(filename, 'w+') as f:
    for i in range (int(num)):
        f.write('A')
        i = i + 1

f.close()

# If previous operation completed then press Enter to exit
print("Operation Completed. Press Enter to exit.")
input()

File: TypeScript/Printcharacter.ts
// SPDX-FileCopyrightText: Copyright (c) 2021 Sam W
// SPDX-License-Identifier: MIT
/**
 * Copyright (c) 2021 Sam W
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to
 * deal in the Software without restriction, including without limitation the
 * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
 * sell copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in
 * all copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 * IN THE SOFTWARE.
 */

import readline from 'readline';
import { open, close } from 'fs';

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let num: number;
let filename: string;

rl.question("How many 'A' should be written to a file: ", (answer: number) => {
    num = answer;
    rl.close();
});

rl.question("What is the name for the file: ", (answer: string) => {
    filename = answer;
    rl.close();
});

open(filename: string, 'wx',)